rudder-webapp/SOURCES/rudder-metrics-reporting

```diff
@@ -143,6 +146,14 @@ def metrics():
   cmd="[ -x /usr/bin/dpkg ] && dpkg-query -W rudder-server-root || rpm -q rudder-server-root"
   data["package_version"] = command(["/bin/sh", "-c", cmd]).strip()
 
+  # Report mode
+  cmd="grep 'RUDDER_REPORT_MODE' /var/rudder/cfengine-community/inputs/rudder.json | sed 's/.*"RUDDER_REPORT_MODE":"\(.*\)",.*/\1/'"
```

Comment:
this regex work with current json, but not necessarily if formatting change
I would remove the coma and add optional spaces around the semicolon

Reply:
should we load rudder.json as a python object instead?

Reply:
this is exactly the regexp used in rudder agent info. 

@amousset do you thinkg we would really benefit from reading the json ?

Reply:
ok